packages/storybook-block/src/utils/buildIframeUrl.spec.ts

```diff
@@ -34,7 +34,7 @@ describe('buildIframeUrl', () => {
             StorybookPosition.Horizontal
         );
         expect(result?.toString()).toEqual(
-            'https://fondue-components.frontify.com/iframe.html?path=%2Fstory%2Fcomponents-action-menu--action-menu&nav=false&panel=false'
+            'http://fondue-components.frontify.com/iframe.html?path=%2Fstory%2Fcomponents-action-menu--action-menu&nav=false&panel=false'
```

Comment:
I would remove this test since it doesn't make any sense anymore.
_It should set an http url to https_ --> it does not change the protocol anymore